x-pack/plugins/observability/public/pages/overview/data_sections.tsx

```diff
@@ -64,7 +64,8 @@ export function DataSections({ bucketSize, hasData, absoluteTime, relativeTime }
             />
           </EuiFlexItem>
         )}
-        {hasData?.ux && (
+        {/* @ts-ignore see https://github.com/elastic/kibana/issues/81081 */}
+        {hasData?.ux?.hasData && (
```

Comment:
instead of changing the api or suppressing the type , i feel like we can improve the types here. if you notice two lines below it's also `(hasData.ux as UXHasDataResponse).serviceName as string` used like that.
there is actually already a type declared to handle this specific api use case https://github.com/elastic/kibana/blob/master/x-pack/plugins/observability/public/typings/fetch_overview_data/index.ts#L40

need to check why it's working.

Reply:
Great suggestion!